### ISSUE TITLE
Increment snake size when head meets energy blocks.

### ISSUE THREAD

**Issue by monacofj** [courselab/ttsnake#6]
This depends on #5 and #3.

The snake grows by adding a body block right after the head (how about that?).

**Comment by JayroBoy** [comment]
@monacofj Due to how the snake is implemented, I figured it was easier to add the new piece to the body of the snake as its tail, right after it moves. This causes a visual effect of the tail remaining in place but the snake stretching in size. Just getting back to you because you suggested that the new block be added to the middle of the body, (right after the head), and from a visual perspective that's what I tried to go for.

**Comment by monacofj** [comment]
> @monacofj Due to how the snake is implemented, I figured it was easier to add the new piece to the body of the snake as its tail, right after it moves. This causes a visual effect of the tail remaining in place but the snake stretching in size. Just getting back to you because you suggested that the new block be added to the middle of the body, (right after the head), and from a visual perspective that's what I tried to go for.

@JayroBoy  Well, it's valid. The reason I had suggested to add the new piece at the head and not after the tail is because it might came out less intuitive. If the a long snake eats a cooky somewhere on the lattice, its tail somewhere else might touch the border... 

But this may be fixed in a new issue.
